### PR TITLE
added commander, restructure files ..

### PR DESCRIPTION
**Pull request by bomsy**
Still work in progress

- Added commander
- Restructure files

### CODE REVIEW

File: bin/foo [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  .usage('[command]')
+
+program
+  .command('checkout <identifier>')
+  .alias('co')
+  .description('checks out a remote branch')
+
+program
+  .command('log')
+  .description('log out')
```
Comment (bomsy) [inline bin/foo]: a better description here

File: bin/foo [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+program
+  .command('diff')
+  .description('display diff to the console')
```
Comment (bomsy) [inline bin/foo]: and here

File: bin/foo [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  program
+    .command('push')
+    .description('push')
```
Comment (bomsy) [inline bin/foo]: and here

File: bin/foo [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+program
+  .version(pkg.version)
+
+program
+  .usage('[command]')
+
+program
+  .command('checkout <identifier>')
+  .alias('co')
+  .description('checks out a remote branch')
```
Comment (bomsy) [inline bin/foo]: and maybe here

**Comment by jasonLaster** [comment]
:dancer:

**Pull request merged**